examples/mongodbatlas_flex_cluster/main.tf

```diff
@@ -0,0 +1,26 @@
+resource "mongodbatlas_flex_cluster" "flex_cluster" {
```

Comment:
[nit] I prefer naming it `this` instead of having `flex_cluster` show up twice

Reply:
Good point, replaced `flex_cluster` with `example_cluster`, etc 👍